cloud/aws/alb/monitors-alb.tf

```diff
@@ -0,0 +1,203 @@
+resource "signalfx_detector" "heartbeat" {
+	name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] AWS Alb heartbeat"
+
+	program_text = <<-EOF
+		from signalfx.detectors.not_reporting import not_reporting
+		signal = data('RequestCount', filter=filter('stat', 'sum') and filter('namespace', 'AWS/ApplicationELB') and ${module.filter-tags.filter_custom})
```

Comment:
I am pretty sure the requestcount metric could be not reported if there is not request on loadblancer. If I am right, this could generate false positive and using healthly host count metric could solve this

Reply:
based on docs it's always reported https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-cloudwatch-metrics.html i